### DISCUSSION TITLE
Create a Grafana Federate

### DISCUSSION THREAD

**Discussion by valdemarpavesi** [grafana/grafana#58295] [General]
There are cases in which we do need a Grafana Federate.

![image](https://user-images.githubusercontent.com/10376101/200173748-ddd7e180-9e5d-4484-9f87-b34298add2da.png)

it is similar to:
https://github.com/valdemarpavesi/grafana-load-test

We do have everything available, just find a solution to pass the datasource-name to the second Grafana.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
